### PR TITLE
setting description screen

### PR DESCRIPTION
**Pull request by EntyR**

### CODE REVIEW

File: app/src/main/res/layout/fragment_random_number_description.xml [hunk 1]
```diff
@@ -10,10 +10,10 @@
...
         android:layout_width="0dp"
         android:layout_height="0dp"
-        android:layout_marginStart="@dimen/linear_layout_margins_horizontal"
-        android:layout_marginTop="@dimen/linear_layout_margins_horizontal"
-        android:layout_marginEnd="@dimen/linear_layout_margins_horizontal"
-        android:layout_marginBottom="@dimen/linear_layout_margins_bottom"
+        android:layout_marginStart="12dp"
+        android:layout_marginTop="70dp"
+        android:layout_marginEnd="12dp"
+        android:layout_marginBottom="150dp"
```
Comment (AlexeyKolosov) [inline app/src/main/res/layout/fragment_random_number_description.xml]: please refer to @dimens.xml

File: app/src/main/res/layout/fragment_random_number_description.xml [hunk 2]
```diff
@@ -29,8 +29,10 @@
             android:layout_marginLeft="@dimen/title_description_margins"
             android:layout_marginTop="@dimen/title_description_margins"
             android:layout_marginRight="@dimen/title_description_margins"
-            tools:text="TextView"
-            android:textSize="@dimen/title_description_text_size" />
+            android:fontFamily="@font/roboto_bold"
+            android:textColor="#000000"
```
Comment (AlexeyKolosov) [inline app/src/main/res/layout/fragment_random_number_description.xml]: please refer to @colors.xml

File: app/src/main/res/layout/fragment_random_number_description.xml [hunk 3]
```diff
@@ -39,21 +41,43 @@
...
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent">
+
+        <ImageView
+            android:id="@+id/imageView"
+            android:layout_width="wrap_content"
+            android:layout_height="0dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="5dp"
```
Comment (AlexeyKolosov) [inline app/src/main/res/layout/fragment_random_number_description.xml]: dimens

File: app/src/main/res/drawable/fullscreen_item_shape.xml [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+        <shape>
+            <corners android:radius="35dp" />
+            <gradient
+                android:startColor="#2A000000"
+                android:endColor="@android:color/transparent"
+                android:angle="90" />
+        </shape>
+
+    </item>
+    <item android:bottom="3dp"
```
Comment (AlexeyKolosov) [inline app/src/main/res/drawable/fullscreen_item_shape.xml]: please put all dimens to dimens.xml and refer to @dimens

File: app/src/main/res/layout/fragment_random_number_description.xml (hunk 2 above)
Comment (EntyR) [reply app/src/main/res/layout/fragment_random_number_description.xml]: done

File: app/src/main/res/layout/fragment_random_number_description.xml (hunk 3 above)
Comment (EntyR) [reply app/src/main/res/layout/fragment_random_number_description.xml]: done

File: app/src/main/res/drawable/fullscreen_item_shape.xml (hunk 4 above)
Comment (EntyR) [reply app/src/main/res/drawable/fullscreen_item_shape.xml]: done

**Review by AlexeyKolosov — APPROVED** [review]

**Pull request merged**